internal/provider/protocols/protocols_iscsi_service_resource_test.go

```diff
@@ -0,0 +1,106 @@
+package protocols_test
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	ntest "github.com/netapp/terraform-provider-netapp-ontap/internal/provider"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const nnRegexProtocolsIscsiService string = "iqn\\.1992-08\\.com.netapp:sn.[[:xdigit:]]+:vs.[[:xdigit:]]"
+
+func TestAccIscsiServiceResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { ntest.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: ntest.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Error 2621462: The supplied SVM does not exist
+			{
+				Config:      testAccIscsiServiceResourceConfig("non-existant", true, "alias01"),
+				ExpectError: regexp.MustCompile("Code:\"2621462\""),
+			},
+			// Create and read
+			{
+				Config: testAccIscsiServiceResourceConfig("svm02", true, "alias01"),
```

Comment:
Can you change all the svm names to tf_acc_svm in acc tests here, we are trying to standardize so that it works without interrupting